argyrogounari/assignment1/Question2.java

```diff
@@ -0,0 +1,16 @@
+
+public class Question2 {
+
+	public static void main(String[] args) {
+		
+		SinglyLinkedList<Integer> newList = new SinglyLinkedList<>();
+		newList.addHead(4);
+		newList.addHead(3);
+		newList.addHead(2);
+		newList.addHead(1);
+		newList.addHead(0);
+		newList.findK(4);
```

Comment:
Outputting what the result of newList.findK(4) is would be helpful.
Also, what other values of K could you test?
Do findK(0), findK(5), findK(-1) all work.